rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -385,21 +421,12 @@ def _set_features(
         attribute: Text,
     ) -> None:
         for index, example in enumerate(examples):
-            _sequence_features = Features(
-                sequence_features[index],
-                FEATURE_TYPE_SEQUENCE,
-                attribute,
-                self.component_config[FEATURIZER_CLASS_ALIAS],
-            )
-            example.add_features(_sequence_features)
-
-            _sentence_features = Features(
-                sentence_features[index],
-                FEATURE_TYPE_SENTENCE,
-                attribute,
-                self.component_config[FEATURIZER_CLASS_ALIAS],
+            self.add_features_to_message(
+                sequence=sequence_features[index],
+                sentence=sentence_features[index],
+                message=example,
+                attribute=attribute,
             )
-            example.add_features(_sentence_features)
 
     @classmethod
     def cache_key(
```

Comment:
we no longer need that, right?

Reply:
oh, right, will remove that

Reply:
also removed `provide_context` - which looks like some other component might need a "provide this module" component but if that was the case, we should've found that by now?

Reply:
looked for the code and this constant is not used anywhere and the context is also not persisted as part of the `Interpreter` 🤷🏻